src/client/components/side-drawer/side-drawer.tsx

```diff
@@ -36,6 +36,7 @@ export interface SideDrawerProps extends React.Props<any> {
   isHome?: boolean;
   isCube?: boolean;
   isLink?: boolean;
+  isSettings?: boolean;
```

Comment:
I think this would be better off as a `viewType: "home" | "cube" | "link" | "settings"` variable than a boolean for every option.